sdk/src/main/java/com/mapbox/maps/MapSurface.kt

```diff
@@ -49,6 +52,10 @@ class MapSurface @JvmOverloads constructor(
    */
   fun surfaceCreated() {
     renderer.surfaceCreated()
+    // display should not be null at this point but to be sure we will fallback to DEFAULT_FPS
+    val screenRefreshRate = (weakContext.get()?.getSystemService(Context.WINDOW_SERVICE) as WindowManager)
```

Comment:
Btw, wonder can this weak reference be released here? I mean, could MapSurface be detached and attached to different activity?

Reply:
reverted to strong as strong ref is used in `MapInitOptions` anyway